src/CommandingOfficers/Modifiers/CODamageModifier.java

```diff
@@ -1,37 +1,66 @@
 package CommandingOfficers.Modifiers;
 
+import java.util.ArrayList;
+
+import Terrain.Environment;
 import Units.UnitModel;
 import CommandingOfficers.Commander;
+import Engine.Combat.CombatParameters;
 
-public class CODamageModifier implements COModifier
+// Provides a damage boost either universally, on one terrain
+public class CODamageModifier extends COModifier
 {
   private int attackModifier = 0;
+  private Environment validTerrains = null;
```

Comment:
Why is `validTerrains` plural if there's only one?
Also, is it about validity, or applicability/relevancy/some other word that would make more semantic sense?